config/crd/bases/_.yaml

```diff
@@ -0,0 +1,16 @@
+
+---
+metadata:
+  creationTimestamp: null
+spec:
+  group: ""
+  names:
+    kind: ""
+    plural: ""
+  scope: ""
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: null
+  storedVersions: null
```

Comment:
Is this file correct?

Reply:
how did that get in there. No, this file should not be in here.